packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -250,35 +260,7 @@
 	// except for the inserter on the left
 	@include break-medium() {
 		&.is-fixed {
-
```

Comment:
So much red. Glorious.

Reply:
This was so painful to write I almost fel sorry ahahhaha

Reply:
For what it's worth, this behavior feels pretty solid now, and knowing the mixin has been used for the main editor window for years now should also boost confidence here, IMO. In fact, I hope to mock up a version of this for mobile next week :)